code/modules/telesci/gps.dm

```diff
@@ -45,6 +46,7 @@ var/list/SPS_list = list()
 
 /obj/item/device/gps/emp_act(severity)
 	emped = TRUE
+	transmitting = FALSE //turns off transmission when emp'd
```

Comment:
This comment couldn't be more superfluous.
What this line of code is _doing_, though, needs some consideration:
As things are right now before this PR, after 30 seconds the GPS will re-start reporting its coordinates to other GPSs, but after this PR it will need manual intervention after the 30 seconds have passed.
This also makes the `if(emped)` check in `proc/get_location_name()` dead code, I suppose.

Reply:
yeah, the comment should have been taken out. and yes I did mean for the GPS to require manual intervention to reactivate, as it is mimicking the same behavior used by headsets when they get emped. this intention is stated in the opening post. if this is unfavorable I’m willing to hear a case against it.
I’ll need some time to review the code. I’m not too experienced with this.

Reply:
Alright, I personally don't mind this and it's good to know it's intended behavior. Still, fix the now unreachable code.

Reply:
Checking the diff on mobile is painful, so I'm not sure whether or not you changed this behavior in a later commit, but if you didn't, you should mention this in the changelog